specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMakerRuntime.json

```diff
@@ -75,7 +75,7 @@
           "Knowledgebases"
         ],
         "summary": "Train call to add suggestions to the knowledgebase.",
-        "operationId": "Knowledgebase_Train",
+        "operationId": "Runtime_Train",
```

Comment:
Just a heads up, this will be a breaking changing in SDK.

Reply:
Got it. This is recently added json and not used in SDK. It is just used for docs.